### ISSUE TITLE
Flutter project cannot run on iOS simulator, but can run on real device.

### ISSUE THREAD

**Issue by Edward608** [flutter/flutter#23392]
Both `flutter run --debug` and Run in Xcode shows the same error.

```
ld: targeted OS version does not support use of thread local variables in l1862 for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

**Comment by zoechi** [comment]
Please add the output of `flutter doctor -v`.

**Comment by Edward608** [comment]
[✓] Flutter (Channel master, v0.10.2-pre.30, on Mac OS X 10.14 18A391, locale en-HK)
    • Flutter version 0.10.2-pre.30 at /Users/edwardwong/flutter
    • Framework revision b63ced55b4 (4 hours ago), 2018-10-22 19:20:17 -0700
    • Engine revision 4c79e423dc
    • Dart version 2.1.0-dev.7.1.flutter-45f9462398

[✓] Android toolchain - develop for Android devices (Android SDK 28.0.3)
    • Android SDK at /Users/edwardwong/Library/Android/sdk
    • Android NDK location not configured (optional; useful for native profiling support)
    • Platform android-28, build-tools 28.0.3
    • ANDROID_HOME = /Users/edwardwong/Library/Android/sdk
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1136-b06)
    • All Android licenses accepted.

[✓] iOS toolchain - develop for iOS devices (Xcode 10.0)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 10.0, Build version 10A255
    • ios-deploy 1.9.2
    • CocoaPods version 1.5.3

[✓] Android Studio
    • Android Studio at /Applications/Android Studio 3.3 Preview.app/Contents
    ✗ Flutter plugin not installed; this adds Flutter specific functionality.
    ✗ Dart plugin not installed; this adds Dart specific functionality.
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1136-b06)

[✓] Android Studio (version 3.2)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 29.0.2
    • Dart plugin version 181.5540.11
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1136-b06)

[✓] IntelliJ IDEA Ultimate Edition (version 2018.2.5)
    • IntelliJ at /Applications/IntelliJ IDEA.app
    • Flutter plugin version 29.1.3
    • Dart plugin version 182.4999

[!] VS Code (version 1.28.2)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension not installed; install from
      https://marketplace.visualstudio.com/items?itemName=Dart-Code.flutter

[✓] Connected device (2 available)
    • Edward’s iPhoneX • 8c0a6d19b6856c4e927e583deeeedc710c651506 • ios • iOS 12.0.1
    • iPhone XS Max    • C2F247F8-F6BC-433F-AED4-69CD1F2B3A9A     • ios • iOS 12.0 (simulator)

! Doctor found issues in 1 category.

**Comment by Edward608** [comment]
The issue if fixed by changing the Deployment Target from 8.0 to 9.0 in Xcode. It is a weird one.

**Comment by bhanuka96** [comment]
> The issue if fixed by changing the Deployment Target from 8.0 to 9.0 in Xcode. It is a weird one.

still got this error on 10.0 in Xcode
